packages/myst-parser/tests/myst.spec.ts

```diff
@@ -85,8 +85,10 @@ function stripPositions(tree: GenericParent) {
   return tree;
 }
 
-function removeTightAnnotations(tree: GenericParent) {
+function fixMystDirectives(tree: GenericParent) {
   selectAll('mystDirective', tree).forEach((node) => {
+    // Node markdown is trimmed
+    (node as any).value = (node as any).value?.trim();
```

Comment:
Not totally clear why we need this, without digging into it...? Is there, like, some test-case round-tripping that does not work any more?

Reply:
There are some of these defined in the myst-spec!

Reply:
Yeah, ok - so the `trim()` originally came from respecting those tests, now we no longer totally align with them. (Well, there are already a few ways we don't align with them, this is just one more, not a big deal...)